src/System.Text.Json/tests/System.Text.Json.Tests.csproj

```diff
@@ -19,6 +23,24 @@
     <Compile Include="JsonTestHelper.cs" />
     <Compile Include="JsonWriterStateTests.cs" />
     <Compile Include="ResizableArray.cs" />
+    <Compile Include="Serialization\Array.ReadTests.cs" />
+    <Compile Include="Serialization\Array.WriteTests.cs" />
+    <Compile Include="Serialization\CyclicTests.cs" />
+    <Compile Include="Serialization\EnumTests.cs" />
+    <Compile Include="Serialization\Null.ReadTests.cs" />
+    <Compile Include="Serialization\Null.WriteTests.cs" />
+    <Compile Include="Serialization\PolymorphicTests.cs" />
+    <Compile Include="Serialization\PropertyVisibilityTests.cs" />
+    <Compile Include="Serialization\SpanTests.cs" />
+    <Compile Include="Serialization\Stream.ReadTests.cs" />
+    <Compile Include="Serialization\Stream.WriteTests.cs" />
+    <Compile Include="Serialization\String.ReadTests.cs" />
+    <Compile Include="Serialization\String.WriteTests.cs" />
+    <Compile Include="Serialization\TestClasses.cs" />
+    <Compile Include="Serialization\TestClasses.Polymorphic.cs" />
+    <Compile Include="Serialization\TestData.cs" />
+    <Compile Include="Serialization\Value.ReadTests.cs" />
```

Comment:
nit: Maybe these should be `ReadTests.Value`, `Stream`, `Array`, etc. and `WriteTests.Value`, `Stream`, `Array`, etc. rather than the other way around.

Reply:
I found it more convinient to keep the Value tests next to eachother (read\write) and they are actually the same class as well.